### DISCUSSION TITLE
How to not Allow Some Matrices Together

### DISCUSSION THREAD

**Discussion by erayerdin** [community/community#25515] [Actions]
<p><a href="https://stackoverflow.com/questions/59230600/how-to-not-allow-some-matrices-together" rel="nofollow noopener">The actual question</a>.</p>
<p>Is there a way to ban some matrix combinations in Github Actions?</p>
<p>The problem is described better on StackOverflow, so I gave link to that. The answers on both platforms are welcome.</p>

**Comment by nickelc** [comment, marked as answer]
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix" rel="nofollow noopener">Example excluding configurations from a matrix</a> </p>
<p>This should work.</p>
<pre><code>matrix:
  python-version: [3.5, 3.6, 3.7]
  django-version: [1.11, 2.0, 2.1, 2.2]

  exclude:
    - python-version: 3.7
      django-version: 1.11
</code></pre>

**Comment by erayerdin** [comment]
<p>What if I have to exclude multiple versions, such as excluding:</p>
<p> - py3.7 and dj1.11</p>
<p> - py3.8 and dj1.11</p>
<p>Apparently, <code>python-version: [3.7, 3.8]</code> will not work.</p>
<p>Edit: Okay okay, I think I can manage it with <code>if</code>.</p>

**Comment by nickelc** [comment]
<p>Just add another exclude for <code>python3.8</code></p>
<pre><code>matrix:
  python-version: [3.5, 3.6, 3.7, 3.8]
  django-version: [1.11, 2.0, 2.1, 2.2]

  exclude:
    - python-version: 3.7
      django-version: 1.11
    - python-version: 3.8
      django-version: 1.11
</code></pre>
